### DISCUSSION TITLE
[SOLVED] Is it possible to associate keyboard shortcut to load (Heading/Title) link present in the browser tab?

### DISCUSSION THREAD

**Discussion by Starboy-Xo** [martinrotter/rssguard#589] [Q&A] [answered]
**First, Kindly take a look at the Red Squares.**

![image](https://user-images.githubusercontent.com/15247707/149721695-7a8f2dff-22e8-492c-97ea-8791cd1c205a.png)

Currently, when required, we click on the URL button to load more information of the feed (it loads the website for you, which is great).
Since, this link is attached to the (Heading/Title). Is it possible to associate the (Heading/Title) links with a custom keyboard shortcut. 
If yes, can we have that feature in next update?

**Comment by martinrotter** [comment, marked as answer]
not currently there, but can be added, converting to ticket

**Comment by martinrotter** [comment]
https://github.com/martinrotter/rssguard/issues/590
